cmd/tealdbg/local.go

```diff
@@ -524,20 +524,38 @@ func (r *LocalRunner) RunAll() error {
 
 	failed := 0
 	start := time.Now()
-	for _, run := range r.runs {
-		r.debugger.SaveProgram(run.name, run.program, run.source, run.offsetToLine, run.states)
+	pooledApplicationBudget := uint64(0)
+	credit, _ := transactions.FeeCredit(r.txnGroup, r.proto.MinTxnFee)
+	// ignore error since fees are not important for debugging in most cases
 
+	evalParams := make([]logic.EvalParams, 0, len(r.runs))
```

Comment:
Maybe make it full size from the start, and put them in at index, instead of using append.

I know there's no efficiency gain here, since you allocated len(r.runs).  But this way we always prefer the style of creating full sized array and putting things in, rather than using append.  Makes it less likely the code will be copied to a place that does the inefficient grow thing.

Reply:
done